code/modules/reagents/reagent_containers/cups/_cup.dm

```diff
@@ -18,6 +18,7 @@
 	var/gulp_size = 5
 	///Whether the 'bottle' is made of glass or not so that milk cartons dont shatter when someone gets hit by it.
 	var/isGlass = FALSE
+	var/static/list/transfer_sound = list('sound/effects/liquid_pour1.ogg', 'sound/effects/liquid_pour2.ogg', 'sound/effects/liquid_pour3.ogg')
```

Comment:
Move this list to the line 121(where you call play_sound).

It's only used there locally so it doesn't need to be global